tox.ini

```diff
@@ -9,6 +9,7 @@ commands =
     coverage report --show-missing --fail-under 100
     pre-commit install -f --install-hooks
     pre-commit run --all-files
+    mypy --config-file mypy.ini docker_push_latest_if_changed.py testing
```

Comment:
if you want, you can run `mypy` via `pre-commit` (I'd suggest it too :D) https://github.com/pre-commit/mirrors-mypy